src/components/ContentPage/index.tsx

```diff
@@ -16,13 +17,14 @@ export const HowItWorks = ({ handleClick }: ContentPages) => (
   <article>
     <h1>How it works</h1>
 
-    <section className="drawer" id="what-is-slowtrade" onClick={handleClick}>
-    <h3>What is slow.trade</h3>
-    <span>
-      <p>
-        Slow.trade is a graphical interface and trading platform (the "<strong>Platform</strong>"), that lets you seamlessly interact with the DutchX Decentralized Trading Protocol (the "<strong>DutchX Protocol</strong>").
-      </p>
-    </span>
+    <section className="drawer active" id="what-is-slowtrade" onClick={handleClick}>
+      <h3>What is slow.trade</h3>
+      <span>
+        <p>
+          Slow.trade is a graphical interface and trading platform (the "<strong>Platform</strong>"), that lets you seamlessly interact with the DutchX Decentralized Trading Protocol (the "<strong>DutchX Protocol</strong>").
+        </p>
+        <a href={dutchXSlowtrade} target="_blank"><img src={dutchXSlowtrade} /></a>
```

Comment:
`<a href='assets/content/dutchX_slowtrade.png target="_blank"><img src='assets/content/dutchX_slowtrade.png /></a>`
are you sure?

Reply:
maybe also make it a `Link` if transitioning within the app